specs/waku/waku-1.md

```diff
@@ -309,10 +314,17 @@ Each node MAY decide to whitelist, i.e. do not rate limit, selected IPs or peer
 
 If a peer exceeds node's rate limits, the connection between them MAY be dropped.
 
-Each node SHOULD broadcast its rate limits to its peers using the rate limits packet. The rate limits MAY also be sent as an optional parameter in the handshake.
+Each node SHOULD broadcast its rate limits to its peers using the `status-update` packet. The rate limits MAY also be sent as an optional parameter in the handshake.
 
 Each node SHOULD respect rate limits advertised by its peers. The number of packets SHOULD be throttled in order not to exceed peer's rate limits. If the limit gets exceeded, the connection MAY be dropped by the peer.
 
+Two rate limits strategies are applied:
+
+1) Number of packets per second
+2) Size of packets (in bytes) per second
+
+Both strategies SHOULD be applied per IP address, peer id and topic.
+
```

Comment:
In the implementation if a single packet has a size greater than the limit, should that packet be dropped or chunked?

Reply:
Thats documented in the "Maximum Packet Size" section @Samyoul

Reply:
I would leave that to the client, currently we don't support chunking, so the packet would just be dropped. I would also expect nodes to set `Limit >= MaxPackeSize`, so that the maximum packet size is properly communicated.

Reply:
Ok, so dropped, if the packet exceeds the maximum packet size.

If the rate limit is 0.5 mb per second and the maximum packet size is 1 mb, would a 0.75 mb packet also be dropped?

Reply:
I would expect something on those lines, whether the packet is actually dropped is up to the client, I think the specs say that a client "MAY" be disconnected

Reply:
Should we include a recommendation along the lines of what you presume?:

> expect nodes to set Limit >= MaxPackeSize, so that the maximum packet size is properly communicated.

Reply:
Sure, I think it makes sense

Reply:
added